packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -233,6 +233,7 @@ Future<String> _buildAotSnapshot(
         '--instructions_blob=$instructionsBlob',
         '--embedder_entry_points_manifest=$vmEntryPointsAndroid',
         '--no-sim-use-hardfp',
+        '--no-use-integer-division',  // Not supported by the Pixel in 32-bit mode.
```

Comment:
Currently this is already being done by gen_snapshot's main.  But it seems prudent to disable it explicitly in case gen_snapshot's defaults change